Storage/SQL/RemoteTabsTable.swift

```diff
@@ -20,6 +20,7 @@ class RemoteClientsTable<T>: GenericTable<RemoteClient> {
             "type TEXT",
             "formfactor TEXT",
             "os TEXT",
+            "fxaDeviceId TEXT",
```

Comment:
Yeah, I actually didn't look at this change at all, so thanks for pointing it out.